### PR TITLE
Prepare 1.2.1 release

### PR DESCRIPTION
**Pull request by hugovk**
And move changelog to own file.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+- Improve error checking when parsing a Blurb.
+- Loosen README check for CPython forks.
+- Move code from `python/core-workflow` to own `python/blurb` repo.
+- Deploy to PyPI via Trusted Publishers.
+
+## 1.1.0
+
+- Support GitHub Issues in addition to b.p.o (bugs.python.org).
+  If "gh-issue" is in the metadata, then the filename will contain
+  "gh-issue-<number>" instead of "bpo-".
```
Comment (ezio-melotti) [inline CHANGELOG.md:24]:
```suggestion
  If `gh-issue` is in the metadata, then the filename will contain
  `gh-issue-<number>` instead of `bpo-`.
```

**Review by ezio-melotti — APPROVED** [review]

**Comment by larryhastings** [comment]
Have you considered using an external tool to maintain the NEWS entries, instead of maintaining it by hand?

**Comment by hugovk** [comment]
Yes, we're actually also using GitHub's own tool to generate release notes ([documented here](https://github.com/python/blurb/blob/main/RELEASING.md)), and they're shown on the release pages here:

* https://github.com/python/blurb/releases

In my own projects, I'm using a similar, external tool, https://github.com/release-drafter/release-drafter. I'm mainly still using this because it predates GitHub's own tool, and has some more options, and I've just not looked into switching yet.

But I think we might be okay with just the GitHub tool here. One downside is that generated changelog isn't part of the Git repo, but to be honest, I've not had any issues or requests for something to be in repo in my own projects. In one, I "retired" the changelog and pointed to the releases page instead: https://github.com/jazzband/prettytable/blob/main/CHANGELOG.md

Shall we just use GitHub's generated release notes?

**Comment by larryhastings** [comment]
I was hinting that we could use blurb itself to maintain blurb's changelog.

**Comment by hugovk** [comment]
That's what I thought at first, then I interpreted "external" as not blurb :)

Sure, we can do, if someone would like to volunteer to set it up. It would be a good way of testing by dogfooding.

**Comment by zooba** [comment]
I'm ready for this change whenever you are! (And by that I mean we're blocked from releasing 3.13.0b4 until this update happens 😉 )

**Comment by hugovk** [comment]
Will release right away!

**Pull request merged**